budget/app.py

```diff
@@ -27,18 +27,14 @@ def get_users(teams):
 
   Returns a dictionary of users with a list of their team memberships
   '''
+  synapseclient.core.cache.CACHE_ROOT_DIR = '/tmp/.synapseCache'
```

Comment:
would it make sense to set this globally?  maybe move it next to log.setLevel()?